### PR TITLE
[Feature] add permission information and fix import problems

### PR DESCRIPTION
**Pull request by kayzliu**
## Description

1. added usage permission information in fake news dataset,
2. added import in init file,
3. fixed specifying pytorch as backend,
4. removed external package usage.
@zhjwy9343 

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: python/dgl/data/fraud.py [hunk 1]
```diff
@@ -262,10 +241,9 @@ class FraudYelpDataset(FraudDataset):
     >>> label = dataset.ndata['label']
     """
 
-    def __init__(self, raw_dir=None, random_seed=2, train_size=0.7, val_size=0.1):
+    def __init__(self, raw_dir=None, train_size=0.7, val_size=0.1):
```
Comment (hetong007) [inline python/dgl/data/fraud.py:244]: Do we not want `random_seed` now?

File: python/dgl/data/fakenews.py [hunk 2]
```diff
@@ -30,6 +29,9 @@ class FakeNewsDataset(DGLBuiltinDataset):
         spacy: the 300-dimensional node feature composed of Twitter user
                historical tweets encoded by the spaCy word2vec encoder.
 
+    Note: this dataset is for academic use only, commercial use requires
+    the approval from Twitter.
```
Comment (hetong007) [inline python/dgl/data/fakenews.py:33]: Can we add link to further details?

**Review by hetong007 — COMMENTED** [review]
For these new datasets, we may want to have test coverage in https://github.com/dmlc/dgl/blob/master/tests/compute/test_data.py

File: python/dgl/data/fakenews.py (hunk 2 above)
Comment (hetong007) [reply python/dgl/data/fakenews.py:33]: Then let's leave the author's contact other than `Twitter`.

File: python/dgl/data/fraud.py (hunk 1 above)
Comment (hetong007) [reply python/dgl/data/fraud.py:244]: @zhjwy9343 please discuss or justify before resolve the conversation.

File: python/dgl/data/fakenews.py [hunk 3]
```diff
@@ -30,6 +29,9 @@ class FakeNewsDataset(DGLBuiltinDataset):
         spacy: the 300-dimensional node feature composed of Twitter user
                historical tweets encoded by the spaCy word2vec encoder.
 
+    Note: this dataset is for academic use only, commercial use requires
```
[the remark this answers is not in the document]
Comment (kayzliu) [reply python/dgl/data/fakenews.py:32]: Added in the latest commit

File: python/dgl/data/fraud.py (hunk 1 above)
Comment (kayzliu) [reply python/dgl/data/fraud.py:244]: Modified based on your comments in the latest commit.

**Pull request merged**